src/openrct2/ride/Ride.cpp

```diff
@@ -2326,6 +2330,7 @@ static void ride_breakdown_update(Ride* ride)
 {
     if (gCurrentTicks & 255)
```

Comment:
MAX_RIDES?

Reply:
I wanted to keep the replays functional, its also not 100% clear what this is supposed to achieve, it ticks updates 254 times leaving only the first tick out. In another test run the vehicles got stuck mid way for some reason, so its for another time.

Reply:
I am not sure 255 is representing `MAX_RIDES` here, it is just trying to make this method run once per 256 ticks. Similar to the every 8192 ticks below.

Reply:
It runs whenever value > 0, but I can see how one can be fooled by this line, it got me too.

Reply:
I did not notice the return, what Ted says.